Provide details as to what each MCD healthcheck indicates (#1175)
kubecost/docs: install-and-configure/install/multi-cluster/multi-cluster-diagnostics.md

![Kubecost-Agent-Diagnostics](/images/diagrams/Agent-Diagnostics-Architecture.png)

## Health Check Definitions

The API response includes several health checks that validate different aspects of your Kubecost deployment. If you see repeated failures for any health check, there is potential for data loss. Please refer to the table below to resolve the issue.

| Health Check | Description |
|--------------|-------------|

> | `kubecostEmittingMetric` | Validates that Kubecost is collecting and emitting metrics for container allocations, network egress, labels, and more ([docs](/architecture/user-metrics.md#kubecost-cost-model)). If failing, try restarting the `kubecost-cost-analyzer` pod. Try port-forwarding to the pod and hitting `http://localhost:9003/metrics` to verify that metrics are being emitted. Reach out to Kubecost support for additional assistance. |

Reviewer: ```suggestion
| `kubecostEmittingMetric` | Validates that Kubecost is collecting and emitting metrics for container allocations, network egress, labels, and more. See the documentation [here](/architecture/user-metrics.md#kubecost-cost-model) for more details. If failing, try restarting the `kubecost-cost-analyzer` pod. Try port-forwarding to the pod and hitting `http://localhost:9003/metrics` to verify that metrics are being emitted. Reach out to Kubecost support for additional assistance. |
```

Author: Thanks!

| `dailyAssetEtlHealthy` | Validates that Kubecost has successfully built Assets ETL files. These ETL files consist of Assets data for the cluster, and are built by querying Prometheus metrics. If failing, refer to the [ETL Repair Guide](/troubleshooting/etl-repair.md) and try repairing the ETL files for the specific cluster. Reach out to Kubecost support for additional assistance. |
| `kubecostPodsNotOOMKilled` | Validates that no pods in the namespace Kubecost is deployed to have OutOfMemoryKilled errors. If failing, check all pods in the namespace Kubecost is deployed to. Review logs of the failing pod. Reach out to Kubecost support for additional assistance. |
| `kubecostPodsNotPending` | Validates that no pods in the namespace Kubecost is deployed to are in a Pending state. If failing, check all pods in the namespace Kubecost is deployed to. Run a `kubectl describe` on all Deployments and StatefulSets to understand the controller events. Reach out to Kubecost support for additional assistance. |

> | `costModelStorageStats` | Validates that PersistentVolume utilization is below 80%. If failing, increase the `PersistentVolume` size. |

Reviewer: ```suggestion
| `costModelStorageStats` | Validates that PersistentVolume utilization is below 80%. If failing, increase the PersistentVolume size. |
```

```
{% endswagger-response %}
{% endswagger %}

> ## API Response Health Checks

Reviewer 2: ```suggestion
## Health Check Definitions
```

Reviewer 2: I'm thinking we may want to put this above the long json as it is more meaningful to the general user. Good with either. and thanks as always for doing this!

Author: Good idea! I'll make the change.
